tools/c7n_azure/c7n_azure/resources/appserviceplan.py

```diff
@@ -26,18 +27,49 @@ class AppServicePlan(ArmResourceManager):
 
     :example:
 
-    Find all App Service Plans that are of the Basic sku tier.
+    Resize app service plans with on/off hours and resource tagging
 
     .. code-block:: yaml
 
         policies:
-          - name: basic-tier-plans
+          - name: on-off-hours-resize-sku
```

Comment:
Do we still need this policy?

Reply:
It feels a little complex as the primary example for app service plan.  Maybe we can put a simple one/old one back on top of it as the first one?  Or the complex one can just go live in the examples folder.

Reply:
oic, maybe the name was just changed incorrectly